src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,18 @@
 package practice;
 
-public class CandidateValidator {
+import java.util.function.Predicate;
+import model.Candidate;
 
+public class CandidateValidator implements Predicate<Candidate> {
+
+    @Override
+    public boolean test(Candidate candidate) {
+        return candidate.getAge() >= 35
+                && candidate.getNationality().equals("Ukrainian")
+                && candidate.isAllowedToVote()
+                && (Integer.parseInt(candidate.getPeriodsInUkr()
+                    .substring(candidate.getPeriodsInUkr().indexOf("-") + 1))
+                    - Integer.parseInt(candidate.getPeriodsInUkr()
+                    .substring(0, candidate.getPeriodsInUkr().indexOf("-"))) > 10);
```

Comment:
```suggestion
                    .substring(candidate.getPeriodsInUkr().indexOf("-") + 1))
                    - Integer.parseInt(candidate.getPeriodsInUkr()
                    .substring(0, candidate.getPeriodsInUkr().indexOf("-"))) > 10);
```
we used in this peace of code the same code twice, so we can implement helper method and make your code more readable